IBPSA/Fluid/FixedResistances/Validation/PlugFlowPipes/MSLAIT.mo

```diff
@@ -0,0 +1,491 @@
+                                        color={191,0,0}));
+  connect(res1.port_a, pip1.heatPorts)
+    annotation (Line(points={{78,20},{78,8.4},{77.9,8.4}},   color={191,0,0}));
+  connect(res1.port_b, col1.port_a)
+    annotation (Line(points={{78,40},{78,46},{92,46}}, color={191,0,0}));
+  connect(pip0.heatPorts, res0.port_a) annotation (Line(points={{128.4,-7.9},{
+          135.2,-7.9},{135.2,-8},{142,-8}},
+                                    color={127,0,0}));
+  connect(res0.port_b, col0.port_a)
+    annotation (Line(points={{162,-8},{172,-8}}, color={191,0,0}));
+  connect(col2.port_b, col5.port_b) annotation (Line(points={{-66,74},{-66,132},
+          {-22,132},{-22,106}}, color={191,0,0}));
+  connect(col5.port_b, col4.port_b) annotation (Line(points={{-22,106},{-22,132},
+          {16,132},{16,106}}, color={191,0,0}));
+  connect(col0.port_b, prescribedTemperature.port) annotation (Line(points={{192,-8},
+          {214,-8},{214,-142},{94,-142}},   color={191,0,0}));
+  connect(col3.port_b, prescribedTemperature.port) annotation (Line(points={{38,-48},
+          {214,-48},{214,-142},{94,-142}},    color={191,0,0}));
+  connect(col4.port_b, prescribedTemperature.port) annotation (Line(points={{16,
+          106},{16,190},{214,190},{214,-142},{94,-142}}, color={191,0,0}));
+  connect(m_flow_zero.y,switch. u3)
+    annotation (Line(points={{178,158},{102,158}},     color={0,0,127}));
+  connect(switch.u1,m_flow_p4. y) annotation (Line(points={{102,174},{178,174}},
+                               color={0,0,127}));
+  connect(Point4.m_flow_in,switch. y) annotation (Line(points={{50,130},{50,166},
+          {79,166}},          color={0,0,127}));
+  connect(switch.u2,lessThreshold. y) annotation (Line(points={{102,166},{108,
+          166},{108,132},{123,132}},
+                             color={255,0,255}));
+  connect(lessThreshold.u,m_flow_p4. y) annotation (Line(points={{146,132},{152,
+          132},{152,174},{178,174}},     color={0,0,127}));
+  connect(lessThreshold.y,switch1. u2) annotation (Line(points={{123,132},{108,
+          132},{108,92},{102,92}},
+                              color={255,0,255}));
+  connect(m_flow_p4.y,switch1. u3) annotation (Line(points={{178,174},{156,174},
+          {156,84},{102,84}}, color={0,0,127}));
+  connect(m_flow_zero.y,switch1. u1) annotation (Line(points={{178,158},{160,
+          158},{160,100},{102,100}},
+                              color={0,0,127}));
+  connect(switch1.y, Point5.m_flow_in) annotation (Line(points={{79,92},{78,92},
+          {78,62},{74,62}}, color={0,0,127}));
+  connect(col1.port_b, prescribedTemperature.port) annotation (Line(points={{
+          112,46},{214,46},{214,-142},{94,-142}}, color={191,0,0}));
+  annotation (
+    Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-160,-160},{
+            220,200}})),
+    experiment(
+      StopTime=603900,
+      Tolerance=1e-006,
+      __Dymola_Algorithm="Dassl"),
```

Comment:
Of the 3 models for comparison, only this one explicitly sets a solver choice for Dymola. Should we have this in either none or all of the models?

Reply:
Added this in the other two models as well.